native/src/components/MapView.tsx

```diff
@@ -24,23 +23,22 @@ type MapViewPropsType = {
 }
 
 const textOffsetY = 1.25
-const layerId = 'point'
+const featureLayerId = 'point'
 
 // Has to be set even if we use map libre
 MapboxGL.setAccessToken(mapConfig.accessToken)
 const MapView = ({ boundingBox, featureCollection, currentFeature }: MapViewPropsType): ReactElement => {
   const mapRef = React.useRef<MapboxGL.MapView | null>(null)
   const cameraRef = React.useRef<MapboxGL.Camera | null>(null)
   const [activeFeature, setActiveFeature] = useState<Feature<Point> | null>(currentFeature ?? null)
```

Comment:
Only really minor and not sure about it: Perhaps `selectedFeature` would be better as we use that already most of the time, active is kind of weird I think